### DISCUSSION TITLE
Matrix Notifaction Support

### DISCUSSION THREAD

**Discussion by minecraftchest1** [upptime/upptime#599] [Ideas]
It would be nice if you would add support for notifications using [matrix](https://matrix.org). There are several SDK's available at https://matrix.org/sdks/, or you can connect manually. The spec is at [spec.matrix.org](https://spec.matrix.org).

**Comment by rhizoet** [comment]
As example from uptime-kuma: https://github.com/louislam/uptime-kuma/blob/master/server/notification-providers/matrix.js
